doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
+
+GVT-g is the GPU virtualization solution of Intel, it is open source and
```

Comment:
Change this first sentence to:

    GVT-g is Intel's open source GPU virtualization solution and is up-streamed
    to the Linux kernel.